week-4/Homework/mandatory/2-writers.js

```diff
@@ -54,19 +54,30 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 
+writers.forEach(writer => console.log(`Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and I work as a ${writer.occupation}`));
 
 
 /*
 Exercise 2:
-  Only `console.log()` out the writers who are in their 40s (meaning between 40 and 49)
+  Only console.log() out the writers who are in their 40s (meaning between 40 and 49)
   and not alive anymore. Use the below sentence format:
   "Writer {firstName} {lastName} died at {age} years old."
 */
+let deadWritersBetween40And49 = writers
+                          .filter(writersInfo => writersInfo.age >= 40 && writersInfo.age < 50)

```

Comment:
Excellent use of chaining with `filter` and `forEach` here, Farhana!
Just beware the indentation as the code is really indented to the right.